ga4gh/backend.py

```diff
@@ -678,6 +730,18 @@ def runGetVariant(self, id_):
         jsonString = gaVariant.toJsonString()
         return jsonString
 
+    def runGetFeature(self, id_):
+        """
```

Comment:
Again, try and be consistent with existing commenting patterns. I agree we should add sphinx doc strings to this class at some point, but it should be done in one go, consistently.